tests/checks/integration/test_file.py

```diff
@@ -28,6 +28,38 @@ def test_present_success(self):
         self.assertTrue(service_checks[0]['status'] == AgentCheck.OK)
         self.assert_tags(['expected_status:present', 'actual_status:present'], service_checks[0]['tags'])
 
+    def test_glob_present_success(self):
+        conf = {
+            'init_config': {},
+            'instances': [
+                {'path': '/tmp/*', 'expect': 'present'}
+            ]
+        }
+        self.check = load_check('file', conf, {})
+        self.check.check(conf['instances'][0])
+        metrics = self.check.get_metrics()
```

Comment:
I think this test might want to create a few files in /tmp just to make sure the assumption holds (-:

(Alternatively, add `'/*'` as a pattern, there's pretty much guaranteed to be something in there)